app.py

```diff
@@ -0,0 +1,29 @@
+import flask
+import requests
+import os
+
+app = flask.Flask(__name__)
+
+MY_API_SECRET_KEY = "x95xd2dbxc3x175md5"
+
+BASE_URL = 'https://api.nytimes.com/svc/search/v2/articlesearch.json'
+
+params = {
```

Comment:
Put API Call in index function to happen every time we load the page. Generally, it is good practice but can work like this.